app/controllers/owners/areas_controller.rb

```diff
@@ -25,15 +25,21 @@ def show
 
       respond_to do |format|
```

Comment:
How about adding here (and removing lower)

```suggestion
      qr_code = QrCode.call(area: area, format: request.format.symbol) unless request.format.symbol == :json

      respond_to do |format|
```